android/hardware/src/main/java/quarks/android/hardware/SensorStreams.java

```diff
@@ -16,14 +16,14 @@ Licensed to the Apache Software Foundation (ASF) under one
 specific language governing permissions and limitations
 under the License.
 */
-package quarks.android.hardware;
+package edgent.android.hardware;
```

Comment:
Thanks!  I see there are a couple of renames that I missed:
./android/hardware/src/main/java/quarks/...
./android/topology/src/main/java/quarks/...
./console/servlets/webapp_content/js/ext/sankey_quarks.js
./console/servlets/webapp_content/resources/images/small_quarks.png
./scripts/runhelloquarks.cmd
./scripts/runhelloquarks.sh